async/src/lib/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+        "target": "ES6",
+        "module": "commonjs",
+        "experimentalAsyncFunctions": true
+    },
+    "files": [
```

Comment:
Why not leave it with no "files" property? makes it user for ppl to play with the sample and add new files..